docs/wiki/Fetching latest changes - Scenario 1 Consume library.md

```diff
@@ -16,33 +16,23 @@ The update process is the following:
-
 # 2. Download the library
 
-To download the updated library follow the best option for your scenario, as explained in [Download the library](./Getting%20started%20-%20Scenario%202%20Consume%20library#1-download-the-library)
+To download the updated library follow the best option for your scenario, as explained in [Download the library](./Getting%20started%20-%20Scenario%201%20Consume%20library#1-download-the-library)
 
 # 3. (Optional) Convert library to ARM
 
-In case you aren't using Bicep you need to follow this procedure: [(Optional) Convert library to ARM](./Getting%20started%20-%20Scenario%202%20Consume%20library#2-optional-convert-library-to-arm)
+In case you aren't using Bicep you need to follow this procedure: [(Optional) Convert library to ARM](./Getting%20started%20-%20Scenario%201%20Consume%20library#2-optional-convert-library-to-arm)
 
 # 4. (Optional) Customize modules
 
-There are different options for library's customization:
-- [Option 1 - Library improvement](./Fetching%20latest%20updates#option-1---library-improvement)
-- [Option 2 - Specific requirements](./Fetching%20latest%20updates#option-2---specific-requirements)
-
-The recommendation is to [contribute](./Contribution%20guide) to the public CARML repository so that your updates can improve the public library. In this way your changes will be available on the public library when fetching updates, modules will be already tested with your changes and you won't need to take care of customization on each update.
-
-In some cases, you might need to add to the library company/organization' specifics, that are not sharable with the public CARML repository.
+The backup folder from step 1, can be used to compare customized modules with the ones coming from the latest version and to implement the required changes. For example, the `'Compare selected'` [function](https://vscode.one/diff-vscode/) in Visual Studio Code can be leveraged for that purpose.
 
-In this scenario every time you'll fetch updates from the public CARML repository merge conflicts are expected. You'll have to compare the new public code with your customized one and re-apply your customizations to the updated code.
+If your copy deviates from the upstream version due to customizations you applied to the code, you'll have to re-apply your customizations to the updated code. This process may be automated, by script or CI, if customization tasks are repeatable.
 
-This process can be automated, by script or CI, if customization tasks are repeatable.
+> **Note**: If customizations are general improvements which may be useful for the public, the recommendation is to [contribute](./Contribution%20guide) to the public CARML repository so that your updates can improve the public library. This way, your changes will be already available the next time you will fetch from upstream, modules will be already tested with your changes and you will not need to take care of customization on each update.
```

Comment:
Consider rewording this bit.. maybe something like: 

"your changes will already be available the next time you fetch from upstream, as modules would already been tested, and would not conflict with your customizations."

Reply:
done